rmgpy/statmech/rotationTest.py

```diff
@@ -53,191 +55,191 @@ def setUp(self):
-            self.assertAlmostEqual(Hexp, Hact, delta=1e-4*Hexp)
+        t_list = np.array([300, 500, 1000, 1500, 2000])
+        h_exp_list = np.array(
+            [0.997705, 0.998624, 0.999312, 0.999541, 0.999656]) * constants.R * t_list
+        for temperature, h_exp in zip(t_list, h_exp_list):
+            h_act = self.mode.getEnthalpy(temperature)
+            self.assertAlmostEqual(h_exp, h_act, delta=1e-4 * h_exp)
 
-    def test_getEntropy_classical(self):
+    def test_get_entropy_classical(self):
         """
         Test the LinearRotor.getEntropy() method using a classical rotor.
         """
         self.mode.quantum = False
-        Tlist = numpy.array([300,500,1000,1500,2000])
-        Sexplist = numpy.array([5.28592, 5.79674, 6.48989, 6.89535, 7.18304]) * constants.R
-        for T, Sexp in zip(Tlist, Sexplist):
-            Sact = self.mode.getEntropy(T)
-            self.assertAlmostEqual(Sexp, Sact, delta=1e-4*Sexp)
-    
-    def test_getEntropy_quantum(self):
+        t_list = np.array([300, 500, 1000, 1500, 2000])
+        s_exp_list = np.array([5.28592, 5.79674, 6.48989, 6.89535, 7.18304]) * constants.R
+        for temperature, s_exp in zip(t_list, s_exp_list):
+            s_act = self.mode.getEntropy(temperature)
+            self.assertAlmostEqual(s_exp, s_act, delta=1e-4 * s_exp)
+
+    def test_get_entropy_quantum(self):
         """
         Test the LinearRotor.getEntropy() method using a quantum rotor.
         """
         self.mode.quantum = True
-        Tlist = numpy.array([300,500,1000,1500,2000])
-        Sexplist = numpy.array([5.28592, 5.79674, 6.48989, 6.89535, 7.18304]) * constants.R
-        for T, Sexp in zip(Tlist, Sexplist):
-            Sact = self.mode.getEntropy(T)
-            self.assertAlmostEqual(Sexp, Sact, delta=1e-4*Sexp)
+        t_list = np.array([300, 500, 1000, 1500, 2000])
+        s_exp_list = np.array([5.28592, 5.79674, 6.48989, 6.89535, 7.18304]) * constants.R
+        for temperature, s_exp in zip(t_list, s_exp_list):
+            s_act = self.mode.getEntropy(temperature)
+            self.assertAlmostEqual(s_exp, s_act, delta=1e-4 * s_exp)
 
-    def test_getSumOfStates_classical(self):
+    def test_get_sum_of_states_classical(self):
         """
         Test the LinearRotor.getSumOfStates() method using a classical rotor.
         """
         self.mode.quantum = False
-        Elist = numpy.arange(0, 2000*11.96, 1.0*11.96)
-        densStates = self.mode.getDensityOfStates(Elist)
-        sumStates = self.mode.getSumOfStates(Elist)
-        for n in range(1, len(Elist)):
-            self.assertAlmostEqual(numpy.sum(densStates[0:n]) / sumStates[n], 1.0, 3)
+        e_list = np.arange(0, 2000 * 11.96, 1.0 * 11.96)
+        dens_states = self.mode.getDensityOfStates(e_list)
+        sum_states = self.mode.getSumOfStates(e_list)
+        for n in range(1, len(e_list)):
+            self.assertAlmostEqual(np.sum(dens_states[0:n]) / sum_states[n], 1.0, 3)
```

Comment:
Could you replace `[0:n]` with `[0: n]`?

Reply:
Since this is slicing, the spacing on both sides of the `:` should be equivalent. I think we prefer no spaces.

Reply:
I am using tight style now, like `[0:n + 1]`